src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Properties/DotNetImportsEnumProvider.cs

```diff
@@ -7,7 +7,7 @@
 namespace Microsoft.VisualStudio.ProjectSystem.Properties;
 
 [ExportDynamicEnumValuesProvider("DotNetImportsEnumProvider")]
-[AppliesTo(ProjectCapability.VisualBasic)]
+[AppliesTo(ProjectCapability.CSharpOrVisualBasic)]
```

Comment:
I can see this `DotNetImportsEnumProvider` is used in the ReferencesPage.VisualBasic.xaml; where is it being referenced for the C# references?